packages/astro/src/compiler/transform/postcss-scoped-styles/index.ts

```diff
@@ -30,6 +30,7 @@ export const NEVER_SCOPED_TAGS = new Set<string>([
   'style',
   'title',
   '!doctype',
+  '!DOCTYPE'
```

Comment:
One last place! This probably needs to not be part of this Set, but a separate check.

Reply:
Got this one now as well.